src/main/java/catan/services/util/play/ActionParamsUtil.java

```diff
@@ -64,6 +65,23 @@
         return edgeIdsToBuildOn;
     }
 
+    public List<Integer> calculateChoosePlayerToRobParams(GameUserBean gameUser) {
+        List<Integer> nodeIdsToBuildOn = new ArrayList<Integer>();
+        for (HexBean hex : gameUser.getGame().getHexes()) {
+            if (!hex.isRobbed()) {
+                continue;
+            }
+            for (NodeBean node : hex.getNodes().listAllNotNullItems()) {
+                if (node.getBuilding() != null && !node.getBuilding().getBuildingOwner().equals(gameUser)) {
```

Comment:
should we check for it if we call listAllNotNullItems()? 
`node.getBuilding() != null`